File: 2.0/java/me/findApiCredentials.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.ApiCredentials;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.common.model.request.CommonFindRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("account_login", "account_password");
        CommonFindRequest request = CommonFindRequest.create()
            .limit(2L)
            .build();
        Page<ApiCredentials> credentials = callfireClient.meApi().findApiCredentials(request);
    }
}

File: 2.0/csharp/me/findApiCredentials.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Common.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("account_login", "account_password");
        var request = new CommonFindRequest { Fields = "name,enabled" };
        Page<ApiCredentials> apiCredentials = client.MeApi.FindApiCredentials(request);
    }
}
